files/ioutil.go

```diff
@@ -45,7 +46,12 @@ func download(URL string, destPath string) error {
 	return nil
 }
 
-func copyFile(src string, dest string) error {
+func CopyResource(src string, dest string) error {
+	cmd := exec.Command("cp", "-r", src, dest)
```

Comment:
why external command instead of programmable copy like this  https://opensource.com/article/18/6/copying-files-go ?

Reply:
That is strange. I doubt that we would do the same in java code.
Did you estimate performance difference?